.gitignore

```diff
@@ -1,62 +1,6 @@
 # Specifies intentionally untracked files to ignore when using Git
 # http://git-scm.com/docs/gitignore
 
-# These ones are put in place from src/assets when we run:
```

Comment:
Looking at [.gitignore](https://github.com/EyeSeeTea/FIRE-WiFiCalling/blob/705ac9bb0de0ff51e20942cf42d252ae22f48202/.gitignore) file, I see all the ignores. I didn't remove any of them intentionally, but maybe because I generated the project with ionic, it came with a new updated file